NEWS

```diff
@@ -1,3 +1,7 @@
+1.1.0 -- 31 Oct 2015
+
+Added a dependency on htmlwidgets and 'offline' plots are now the default. If you want to create a figure on a plotly server, you need to use `plotly_POST()`. Also added a `config()` function to control the default appearance of the interactive plot
```

Comment:
IIRC, you moved the docs to knitr/rmarkdown so you should be able to just recompile and have the examples embedded, right?